### PR TITLE
Update setup wizard page and api for usage data

### PR DESCRIPTION
**Pull request by bhalsey**
Update the setup wizard screenshot and corresponding wording.

Update docs for `/api/system-configuration`

### Linked PR:
- https://github.com/FusionAuth/fusionauth-site/pull/3393

### CODE REVIEW

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 1]
```diff
@@ -18,13 +18,15 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
 This will be the first account to be created in FusionAuth and this user will be assigned the `admin` role the FusionAuth application. Additional
 users can be assigned the `admin` role at a later time.
 
-#### License agreement and communications
+#### Analytics & communications
 
-To complete the setup wizard you must toggle the Accept button after reading the FusionAuth license agreement. You won't find any surprise strings attached here, just the normal lawyer speak for CYA.
+Your feedback on how you learned about us helps us better serve our customers.
```
Comment (lyleschemmerling) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:23]:
```suggestion
Your feedback on how you learned about FusionAuth helps us better serve our customers.
```

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 2]
```diff
@@ -18,13 +18,15 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
 
 Optionally stay informed by opting-in to receive FusionAuth technical announcements, tutorials and other technical focused emails. You may opt-out at any time in the future, or opt-in at a later date by visiting [fusionauth.io](/) and finding the "Subscribe for updates" form at the bottom of the page.
 
-Your feedback on how you learned about us helps us better serve our customers.
+If your instance is hosted on FusionAuth Cloud, then usage stats collection is part of the cloud hosting package. Self-hosted customers can disable usage stats collection. See [Collected Metrics](/docs/get-started/download-and-install/collected-metrics) for more information.
```
Comment (lyleschemmerling) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:27]: Are officially cool with `usage stats` in our verbiage or do we want to say `usage statistics`? 🤓

File: astro/src/content/docs/_shared/_setup-wizard.mdx [hunk 3]
```diff
@@ -18,13 +18,15 @@ Once the setup wizard is complete you will be logged into FusionAuth using the c
...
+
+To complete the setup wizard you must toggle the Accept button after reading the FusionAuth license agreement. You won't find any surprise strings attached here, just the normal lawyer speak for CYA.
```
Comment (lyleschemmerling) [inline astro/src/content/docs/_shared/_setup-wizard.mdx:29]:
```suggestion
To complete the Setup Wizard you must toggle the Accept button after reading the FusionAuth license agreement. You won't find any surprise strings attached here, just the normal lawyer speak for CYA.
```

**Review by lyleschemmerling — APPROVED** [review]
minor suggestions

File: astro/src/content/docs/_shared/_setup-wizard.mdx (hunk 2 above)
Comment (bhalsey) [reply astro/src/content/docs/_shared/_setup-wizard.mdx:27]: I'm not sure who decides what's officially cool around here, but we're usings stats a lot in this PR https://github.com/FusionAuth/fusionauth-site/pull/3393/files

**Pull request merged**